BoldyrevEV/analyzer.js

```diff
@@ -0,0 +1,32 @@
+var fs = require('fs');
+
+fs.readFile("text.txt", 'utf8', (err, data) => {
```

Comment:
Постарайтесь использовать только один вид кавычек
```suggestion
fs.readFile('text.txt', 'utf8', (err, data) => {
```